pkg/config/init.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+// InitOptions is the options for the init command
+type InitOptions struct {
+	Force bool
+}
+
+// NewInitOptions creates a new InitOptions
+func NewInitOptions() *InitOptions {
+	return &InitOptions{}
+}
+
+// InitImpl is impl for InitOptions
+type InitImpl struct {
+	*GlobalImpl
+	*InitOptions
+}
+
+// NewInitImpl creates a new InitImpl
+func NewInitImpl(g *GlobalImpl, b *InitOptions) *InitImpl {
+	return &InitImpl{
+		GlobalImpl:  g,
+		InitOptions: b,
+	}
+}
+
+// Args returns the args.
+func (b *InitImpl) Args() string {
+	return ""
+}
```

Comment:
```suggestion
```

Reply:
interface `ConfigProvider` need `Args` method, The `app.New(initImpl)` method in `cmd/init.go` requires `ConfigProvider`

Reply:
Don't need any more in new code.